### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by bluPhy**
Fixing typos

### CODE REVIEW

File: forwarders/schema_utility_test.py [hunk 1]
```diff
@@ -225,7 +225,7 @@ def test_process_input_detailed_schema_non_primitive(
 ========================================
 
 ========================================
-======== Forwarder Cofiguration ========
+======== Forwarder Configuration ========
```
Comment (bumblebee211196) [inline forwarders/schema_utility_test.py:228]: Remove extra "+" at the end.

**Review by bumblebee211196 — COMMENTED** [review]
Please address the comments.

File: forwarders/schema_utility_test.py (hunk 1 above)
Comment (bluPhy) [reply forwarders/schema_utility_test.py]: Remove extra "="

**Review by bumblebee211196 — APPROVED** [review]
LGTM!

**Pull request merged**